src/sharedServices/s3/copyS3Object.ts

```diff
@@ -6,16 +6,17 @@ import {
 import { getEnv } from '../../utils/helpers'
 
 export const copyS3Object = async (
-  fileName: string,
-  fileLocationPath: string,
-  fileDestinationBucket: string
+  Key: string,
```

Comment:
Can you stick to the normal Camel case for function parameters here?

Reply:
done